waspc/data/Generator/templates/sdk/wasp/server/email/index.ts

```diff
@@ -2,27 +2,30 @@
 import { initEmailSender } from "./core/index.js";
 import { EmailSender } from "./core/types.js";
 
+// TODO: We need to validate all the env variables
+// For now, we are letting the runtime throw if they are not provided
 {=# isSmtpProviderUsed =}
 const emailProvider = {
     type: "smtp",
-    host: process.env.SMTP_HOST,
+    host: process.env.SMTP_HOST!,
+    // @ts-ignore
     port: parseInt(process.env.SMTP_PORT, 10),
-    username: process.env.SMTP_USERNAME,
-    password: process.env.SMTP_PASSWORD,
+    username: process.env.SMTP_USERNAME!,
+    password: process.env.SMTP_PASSWORD!,
```

Comment:
This file is throwing all kinds of errors for me. I tried to build the server with TypeScript (`npx tsc` in `.wasp/out/server`) and it failed.

Does this work for you normally (I'm guessing it does, because you wouldn't know these types needed changing if it didn't).

I probably messed something up because it also fails on older versions of main. If everything's normal for you, I'll dig in.